spec/acceptance/listen_spec.rb

```diff
@@ -144,8 +142,7 @@
             it 'listens to file move' do
               expected = { modified: [],
                            added: %w(file.rb),
-                           removed: %w(dir/file.rb)
-              }
+                           removed: %w(dir/file.rb) }
```

Comment:
The same extra indentation could be avoided here as well. Though its upto the maintainer's preference.
```ruby
expected =
  { 
    modified: [],
    added: %w(file.rb),
    removed: %w(dir/file.rb)
  }
```

Reply:
Please propose a change to the `rubocop` rules so it tolerates the formatting you like.

Reply:
I'm not sure why `rubocop -a` does not did that but `Layout/MultilineAssignmentLayout` defaults to new line.
Maybe running `rubocop -a --only Layout/MultilineAssignmentLayout` might work?